### ISSUE TITLE
[enhancement] reorder wcag version filter choices

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-policies-prototype#19]

**Comment by AndrewArch** [comment]
@slhenry - can you elaborate?

**Comment by slhenry** [comment]
e.g., show me all that use WCAG 2

**Comment by sharronrush** [comment]
The use cases that occur to me are these, but I will leave this for MaryJo's much greater experience:
- countries that use WCAG (and sort by level as well) as is
- countries that have use WCAG + additional requirements
- dates of publication
- date of enforcement
- countries that also have procurement requirements

**Comment by bakkenb** [comment]
I don't have comment here. I don't feel I have experience to know what people would search for besides what has been outlined in the Use Cases, by Mary Jo, and by Sharron above.

**Comment by yatil** [comment]
From the February 28th F2F: 

* No filter for the date (as you can sort by date).
* Question if we need a way to filter by multiple countries. Maybe not worth the complexity in the UI.
* Filters update results in table and on the page.
* Filtering only the table and link to individual pages? General agreement when also having a view for all countries.
* Link to individual pages from the table and link to anchors inside of the target country pages.

**Comment by dboudreau** [comment]
I'm thinking the filters for 'type" should also include another option, "mandatory standard", for instances where the adopted policy is not a law, but still an expected requirement for government (as is the case for the province of Quebec).

**Comment by slhenry** [comment]
* There still needs to be some synch-ing between the terms used throughout, for example, in the Filter options, "Standard" should probably be "WCAG Version Used" like in the table.
* I'm not convinced that "Status: Proposed/in development; In effect" is worth including as a filter. A person can sort the table by Date Enacted to get that. Also, it seems not a high usage from the use cases. (But I don't feel strongly about this.)

As I was focusing on the filter options, I was wondering what some of the words meant, and wondering if we wanted to include the description there. "Terms on this page" is above it.
* What about putting the descriptions of the Terms in the Filter area and make it visible by default (and probably optionally collapsible). I think that would improve usability and take up less space! :-)

**Comment by slhenry** [comment]
I think Filtering Options needs clearer heading. Depending on https://github.com/w3c/wai-policies-prototype/issues/56 , then probably outside of "Law & Policy Overview Table" section.

**Comment by bakkenb** [comment]
I am confused by the filter "Status." Would something that is **proposed** or **in development** even be listed on this page? If it is not a policy why would it be listed? Maybe this is for when a law or policy is in effect but not enforced for a year or more??? Just trying to understand what data this filter would be based off of that would trigger it and where else that information would show up on the page. I don't see it anywhere currently in the prototype or on the [existing resource](https://www.w3.org/WAI/Policy/Overview.html).
Sorry if I am missing something.

**Comment by bakkenb** [comment]
Agree with the need to match the filter with the column heading or heading used after the table in the "Laws & Policies by Country" section.

**Comment by nitedog** [comment]
For filter "standard" I suggest adding the value "other".

**Comment by iamjolly** [comment]
Filters have now been added to the prototype. We are opening new issues for any bugs and enhancements. Closing this for now.

**Comment by slhenry** [comment]
Agree with Eric above:

> [strong] If we can’t leave WCAG 1.0 out, WCAG 2.0 should be first.

**Comment by iamjolly** [comment]
@slhenry - This is only a "nice to have" and is not critical for launch. Moving to Post-Launch Enhancements milestone.

**Comment by iamjolly** [comment]
Also, this issue will now be renamed to "[enhancement] reorder wcag version filter choices" to make it clear what is needed.

**Comment by slhenry** [comment]
umm - but changing the filter option order *after* people are used to it, would not be nice. I assume it's a 5 minute fix, and should be done before launch.
